src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -922,216 +910,92 @@ void testErrorReportedDuringRegistrationWhenHasSoldLandFieldIsTrueForPartialVali
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
-    void testErrorReportedDuringRegistrationWhenHasSoldLandFieldIsTrueForFullValidation() throws ServiceException {
-        setIsRedisRemovalEnabledFeatureFlag(true);
-        buildOverseasEntitySubmissionDto();
-        overseasEntitySubmissionDto.setHasSoldLand(Boolean.TRUE);
-
-        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(),
```

Comment:
The full validation never calls these methods so there is no point in adding tests to check that the methods that are not called don't raise an error.

Reply:
I agree with Martin, I think it would be good to have that test as we are testing that the data in that state should pass validation, and if someone would change the behaviour of validation to not allow nulls for those fields then the test would catch it before it goes into production.

Reply:
validateFull never uses the validateHasSoldLand, validateIsSecureRegister and validateWhoIsRegistering methods.
So if the behaviour changes in these methods, it will have no impact, as they are not called from validateFull.

Reply:
The new test would be for the requirement/use case that the full validation should allow nulls for those 3 fields. If someone were to add the null validation for those 3 fields into validateFull (as they are unaware of our discussions) then I don't think a test would fail and it would appear that everything is working as expected.

Reply:
Ok, discussed this. Agreed that the comment in the application code will suffice here.